pkg/buildkit/buildkit.go

```diff
@@ -199,11 +196,11 @@ func (c *Client) Build(opts BuildOptions) error {
 	}
 
 	// build/push images
-	return c.runSolve(solveOpt)
+	return c.runSolve(ctx, solveOpt)
 }
 
 func (c *Client) Cache(image string) error {
-	return c.solveWith(func(buildDir string, solveOpt *bkclient.SolveOpt) error {
+	return c.solveWith(context.TODO(), func(buildDir string, solveOpt *bkclient.SolveOpt) error {
```

Comment:
why not pass this context through the caller even though it's unused?

Reply:
makes sense, i was just being lazy.